VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/LaunchFeatureTest.java

```diff
@@ -0,0 +1,47 @@
+package org.edx.mobile.test.feature;
+
+import android.app.Instrumentation;
+import android.content.Intent;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.edx.mobile.R;
+import org.edx.mobile.base.MainApplication;
+import org.edx.mobile.core.EdxEnvironment;
+import org.edx.mobile.module.prefs.PrefManager;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+@RunWith(AndroidJUnit4.class)
+public class LaunchFeatureTest {
+
+    @Test
+    public void whenAppLaunched_withAnonymousUser_landingScreenIsDisplayed() {
+        {
+            // Ensure we are not logged in
+            final MainApplication application = MainApplication.instance();
+            final EdxEnvironment environment = application.getInjector().getInstance(EdxEnvironment.class);
+            new PrefManager(application, PrefManager.Pref.LOGIN).clearAuth();
+            environment.getSegment().resetIdentifyUser();
+            environment.getNotificationDelegate().unsubscribeAll();
```

Comment:
I think this can be removed. Logout works fine without crashing with the mobile-devi parse config